### PR TITLE
Added Jobs

### PR DESCRIPTION
**Pull request by MishaelPhilip**
Changes:
- Added new jobs

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -119,6 +119,31 @@ <h2>Job Opportunities</h2>
...
+            <tr class="job-infos">
+                <td>Digital Marketing</td>
+                <td>Lagos</td>
+                <td>$40,000</td>
+                <td>Sept 18, 2023</td>
+            </tr>
+            <tr class="job-infos">
+                <td>Social Media Manager</td>
+                <td>Lagos</td>
+                <td>$50,000</td>
```
Comment (MishaelPhilip) [inline src/index.html]: add the salary to 65,000

File: src/index.html [hunk 2]
```diff
@@ -119,6 +119,31 @@ <h2>Job Opportunities</h2>
...
+                <td>July 12, 2023</td>
+            </tr>
+            <tr class="job-infos">
+                <td>Project Manager</td>
+                <td>Lagos</td>
+                <td>$240,000</td>
```
Comment (MishaelPhilip) [inline src/index.html:137]: 280,000

File: src/index.html [hunk 3]
```diff
@@ -119,6 +119,31 @@ <h2>Job Opportunities</h2>
...
+                <td>Dec 12, 2023</td>
+            </tr>
+            <tr class="job-infos">
+                <td>IT Support</td>
+                <td>Abuja</td>
+                <td>$150,000</td>
```
Comment (MishaelPhilip) [inline src/index.html:143]: 170,000

File: src/index.html [hunk 4]
```diff
@@ -119,6 +119,31 @@ <h2>Job Opportunities</h2>
                 <td>$50,000</td>
```
Comment (emmseze) [inline src/index.html:119]: change the salary to 80,000

File: src/index.html [hunk 5]
```diff
@@ -119,6 +119,31 @@ <h2>Job Opportunities</h2>
                 <td>$50,000</td>
                 <td>Sept 18, 2022</td>
             </tr>
+            <tr class="job-infos">
+                <td>Digital Marketing</td>
+                <td>Lagos</td>
```
Comment (emmseze) [inline src/index.html]: put Abuja

File: src/index.html [hunk 6]
```diff
@@ -119,6 +119,31 @@ <h2>Job Opportunities</h2>
...
+                <td>Sept 18, 2023</td>
+            </tr>
+            <tr class="job-infos">
+                <td>Social Media Manager</td>
+                <td>Lagos</td>
+                <td>$50,000</td>
+                <td>July 12, 2023</td>
+            </tr>
+            <tr class="job-infos">
+                <td>Project Manager</td>
```
Comment (emmseze) [inline src/index.html]: Add senior project manager

**Review by emmseze — COMMENTED** [review]
good

**Pull request merged**